runtime/laos/src/apis.rs

```diff
@@ -0,0 +1,381 @@
+#[macro_export]
+macro_rules! impl_runtime_apis_plus {
```

Comment:
why use macro?

Reply:
I tried several implementation that didn't used a macro, noone worked out: it seems the impl_runtime_api macro needs to stay in the root. So ... that's the only solution I worked out following the path of Moonbeam:
https://github.com/moonbeam-foundation/moonbeam/blob/master/runtime/common/src/apis.rs

Reply:
I see. `impl_runtime_apis` implements some traits for the `Runtime` and maybe it can't impl if the `Runtime` is not defined in the current file